components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -860,6 +860,7 @@ void ConversationDriver::SubmitSelectedTextWithQuestion(
         selected_text, question,
         base::BindRepeating(
             [](GeneratedTextCallback received_callback,
+               bool is_delta_text,
```

Comment:
if we were going to handle this here (sorry split across a couple of comments):

```suggestion
               bool is_delta_text, std::string& result,
```